psalm.xml

```diff
@@ -28,7 +28,7 @@
     <issueHandlers>
         <UndefinedClass>
             <errorLevel type="suppress">
-                <referencedClass name="Doctrine\DBAL\Types\Types"/>
+                <referencedClass name="Doctrine\DBAL\Types\Type" />
             </errorLevel>
         </UndefinedClass>
```

Comment:
you removed the usages, so remove this whole block?

Reply:
We use `Doctrine\DBAL\Types\Type` now, so it's still needed.